lib/application.ex

```diff
@@ -387,4 +390,32 @@ defmodule Commanded.Application do
   @doc false
   @spec registry_adapter(Commanded.Application.t()) :: {module, map}
   def registry_adapter(application), do: Config.get(application, :registry)
+
+  @type uuid :: String.t()
+
+  @doc false
+  @spec uuid_generator(Commanded.Application.t(), boolean()) :: (() -> uuid) | nil
+  def uuid_generator(application, true) do
```

Comment:
I would have call this method `uuid_generator!` and removing the last argument

```suggestion
  def uuid_generator!(application) do
```

Reply:
Good suggestion, applied.